### PR TITLE
[Fix] update camera parameter

### PR DESCRIPTION
**Pull request by LazyBusyYang**
* To avoid numpy values in CameraParameter and error caused by them, we add type check to ensure their type.
* Check key in a function
* Fix some docstring in HumanData

### CODE REVIEW

File: mmhuman3d/core/cameras/camera_parameter.py [hunk 1]
```diff
@@ -343,6 +384,70 @@ def get_vibe_dict(self) -> dict:
...
+                key cannot be found in
+                CameraParameter.SUPPORTED_KEYS.
+        """
+        check_passed = True
+        if key not in self.__class__.SUPPORTED_KEYS:
+            check_passed = False
+        if not check_passed:
+            err_msg = 'Key check failed in CameraParameter:\n'
+            err_msg += f'key={str(key)}\n'
+            raise KeyError(err_msg)
```
Comment (pangyyyyy) [inline mmhuman3d/core/cameras/camera_parameter.py:426]:
```
        if key not in self.__class__.SUPPORTED_KEYS:
            err_msg = 'Key check failed in CameraParameter:\n'
            err_msg += f'key={str(key)}\n'
            raise KeyError(err_msg)
```
Is it necessary to store `check_passed`?

File: mmhuman3d/core/cameras/camera_parameter.py [hunk 2]
```diff
@@ -343,6 +384,70 @@ def get_vibe_dict(self) -> dict:
...
+                Value is supported but doesn't match definition.
+        """
+        check_passed = True
+        if type(val) != self.__class__.SUPPORTED_KEYS[key]['type']:
+            check_passed = False
+        if not check_passed:
+            err_msg = 'Type check failed in CameraParameter:\n'
+            err_msg += f'key={str(key)}\n'
+            err_msg += f'type(val)={type(val)}\n'
+            raise TypeError(err_msg)
```
Comment (pangyyyyy) [inline mmhuman3d/core/cameras/camera_parameter.py:449]:
```
        if type(val) != self.__class__.SUPPORTED_KEYS[key]['type']:
            err_msg = 'Type check failed in CameraParameter:\n'
            err_msg += f'key={str(key)}\n'
            err_msg += f'type(val)={type(val)}\n'
            raise TypeError(err_msg)
```
same as above, `check_passed` doesn't seem to be returned or used downstream

File: tests/test_cameras/test_camera_parameter.py [hunk 3]
```diff
@@ -52,6 +52,32 @@ def test_load_json():
     assert len(cam_para.get_value('translation')) == 3
 
 
+def test_type():
+    cam_para = CameraParameter(name='src_cam')
```
Comment (pangyyyyy) [inline tests/test_cameras/test_camera_parameter.py:56]: `cam_param` instead of `cam_para`?

File: tests/test_cameras/test_camera_parameter.py [hunk 4]
```diff
@@ -52,6 +52,32 @@ def test_load_json():
...
+    # wrong distortion value type
+    with pytest.raises(TypeError):
+        cam_para.set_value('k1', '1.0')
+    with pytest.raises(TypeError):
+        cam_para.set_value('k1', np.ones(shape=(3))[0])
+    cam_para.set_value('k1', 1.0)
+    cam_para.reset_distort()
```
Comment (pangyyyyy) [inline tests/test_cameras/test_camera_parameter.py:63]: I think `cam_para.set_value('k1', 1.0)`  would be reset to 0.0 in `cam_para.reset_distort()`, is this intended?

File: mmhuman3d/core/cameras/camera_parameter.py (hunk 1 above)
Comment (LazyBusyYang) [reply mmhuman3d/core/cameras/camera_parameter.py:426]: No, will be removed in hours.

Comment (LazyBusyYang) [reply mmhuman3d/core/cameras/camera_parameter.py]: Fixed in 96daaac.

File: mmhuman3d/core/cameras/camera_parameter.py (hunk 2 above)
Comment (LazyBusyYang) [reply mmhuman3d/core/cameras/camera_parameter.py]: Fixed in 96daaac.

File: tests/test_cameras/test_camera_parameter.py (hunk 3 above)
Comment (LazyBusyYang) [reply tests/test_cameras/test_camera_parameter.py]: Fixed in 96daaac.

File: tests/test_cameras/test_camera_parameter.py (hunk 4 above)
Comment (LazyBusyYang) [reply tests/test_cameras/test_camera_parameter.py]: Yes.

**Review by pangyyyyy — APPROVED** [review]
looks good to me!

**Pull request merged**